vita3k/ctrl/include/ctrl/state.h

```diff
@@ -60,6 +60,7 @@ struct CtrlState {
     ControllerList controllers;
     int controllers_num = 0;
     bool has_motion_support = false;
+    bool drop_inputs = false;
```

Comment:
remove this here
after look code, set this here is totaly not needed, set it inside emuenv state directly